src/components/PWDKDropzone.tsx

```diff
@@ -0,0 +1,254 @@
+  backgroundSize: '1.5px 100%, 100% 1.5px, 1.5px 100% , 100% 1.5px',
+  backgroundPosition: '0 0, 0 0, 100% 0, 0 100%',
+  backgroundRepeat: 'no-repeat',
+  borderRadius: '6px',
+  backgroundColor: colors.neutral800
+}))
+
+// ------------------------------------------------ components
+
+interface PreviewCardProps {
+  fileName: string
+  fileSize: number
+  fileImage: string | undefined
+  onDeleteFileClick: () => void
+}
+
+interface PWDKDropzoneProps {
+  file: File | null
+  accept: string | string[]
+  caption: string
+  maxFileSizeInMb: number
+  error: boolean
+  helperText: string | undefined
+  onChange: (file: File | null) => void
+  title?: string
+}
+
+interface FileError {
+  code: string
+  message: string
+}
+
+const PreviewCard: FC<PreviewCardProps> = ({
+  fileName,
+  fileSize,
+  fileImage,
+  onDeleteFileClick
+}) => {
+  return (
+    <Box
+      position="relative"
+      width="158px"
+      border={`1px solid ${colors.neutral700}`}
+      borderRadius="4px"
+    >
+      <img
+        src={fileImage}
+        alt="Preview"
+        width="100%"
+        height="100%"
+        style={{
+          borderTopLeftRadius: '4px',
+          borderTopRightRadius: '4px',
+          objectFit: 'cover',
+          maxHeight: '70px',
+          display: 'block'
+        }}
+      />
+      <Box p="8px">
+        <Typography typography="caption1" color="neutral100.main">
+          {fileName}
+        </Typography>
+        <Typography typography="captionregular" color="neutral400.main">
+          {fileSizeConverter(fileSize)}
+        </Typography>
+      </Box>
+      <Stack
+        sx={{
+          backgroundColor: 'white',
+          borderRadius: '50%',
+          cursor: 'pointer'
+        }}
+        position="absolute"
+        justifyContent="center"
+        alignItems="center"
+        top="8px"
+        right="8px"
+        width="24px"
+        height="24px"
+        onClick={onDeleteFileClick}
+      >
+        <Close
+          sx={{
+            fontSize: '16px'
+          }}
+        />
+      </Stack>
+    </Box>
+  )
+}
+
+const PWDKDropzone: FC<PWDKDropzoneProps> = ({
+  file,
+  accept,
+  caption,
+  maxFileSizeInMb,
+  error,
+  helperText,
+  onChange,
+  title
+}) => {
+  const inputRef = useRef<HTMLInputElement | null>(null)
+  const [filePreviewUrl, setFilePreviewUrl] = useState<string | undefined>(
+    undefined
+  )
```

Comment:
```suggestion
  const [filePreviewUrl, setFilePreviewUrl] = useState<string>('')
```

bisa pake string kosong sepertinya

Reply:
oiya bisaa